claim/netdata-claim.sh.in

```diff
@@ -264,10 +281,14 @@ HTTP_STATUS_CODE=$(grep "HTTP" "${CLAIMING_DIR}/tmpout.txt" | awk -F " " '{print
 
 if [ "${HTTP_STATUS_CODE}" = "204" ] ; then
         rm -f "${CLAIMING_DIR}/tmpout.txt"
-        echo -n "${ID}" >"${CLAIMING_DIR}/claimed_id"
-        rm -f "${CLAIMING_DIR}/token"
-        echo >&2 "Node was successfully claimed."
-        exit 0
+        echo -n "${ID}" >"${CLAIMING_DIR}/claimed_id" || (echo >&2 "Claiming failed"; set -e; exit 2)
+        rm -f "${CLAIMING_DIR}/token" || (echo >&2 "Claiming failed"; set -e; exit 2)
+        if [ "$EUID" == "0" ]; then
+            chown -R "${NETDATA_USER}:${NETDATA_USER}" ${CLAIMING_DIR}
+        fi
+        netdatacli reload-claiming-state && echo >&2 "Node was successfully claimed." && exit 0
```

Comment:
This script can be called by the agent as well, so it can go like this:
`netdata -W claim -> netdata-claim.sh -> netdatacli -> netdata`
I'm not sure it's safe, we should disable this line when netdata calls the script with a special argument.

Reply:
Not any more - take a look at the change in the .c code.

Reply:
i.e. the script will call reload but I removed the other reload.

Reply:
Ok, now it's not a linear process anymore, netdatacli will spawn a thread asynchronously. Will this process succeed in triggering ACLK?

Reply:
Ah, also this won't work regardless, the command server hasn't been initialized yet so `netdatacli` will fail.

Reply:
Moreover, if for any reason the UNIX socket is not there it wouldn't work as well, it's more robust if the daemon does not rely on `netdatacli`.

Reply:
There have been many recorded cases where the daemon worked fine with a failed command server due to many reasons. However, netdata can work fine with only signals.

Reply:
I am overwhelmed by your sheer weight of argument. Fine fine fine :) I'll put the reload back in the C code and pass a flag instead.